phantazm-zombies-mapeditor/settings.gradle.kts

```diff
@@ -0,0 +1,7 @@
+pluginManagement {
+    repositories {
+        maven("https://maven.fabricmc.net/")
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
```

Comment:
why put a settings file here and not just include this in the root? intellij isn't a huge fan of having the file here